refund/refund.js

```diff
@@ -0,0 +1,249 @@
+require('dotenv').config()
+require('log-timestamp');
+const _ = require('lodash');
+const Kava = require('@kava-labs/javascript-sdk');
+const BnbChain = require("@binance-chain/javascript-sdk");
+const bnbCrypto = BnbChain.crypto;
+
+/**
+ * Automatically refunds any refundable swaps on both Kava and Binance Chain
+ */
+class RefundBot {
+    constructor(bnbChainDeputy, limit = 1000, offsetIncoming = 0, offsetOutgoing = 0) {
+        if (!bnbChainDeputy) {
+            throw new Error("must specify the deputy's Binance Chain address");
+        }
+        this.bnbChainDeputy = bnbChainDeputy
+        this.limit = limit;
+        this.offsetIncoming = offsetIncoming;
+        this.offsetOutgoing = offsetOutgoing;
+    }
+
+  /**
+   * Initialize the Kava client
+   * @param {String} lcdURL api endpoint for Kava's rest-server
+   * @param {String} mnemonic Kava address mnemonic
+   * @return {Promise}
+   */
+    async initKavaClient(lcdURL, mnemonic) {
```

Comment:
Nice init functions 👍 .  If we start using these clients in more bots it'll be easy to extract these into their own functions and inject an already initialized client into bots to share dependencies and make mocking a bit easier down the line.